lib/jsdom/living/hr-time/Performance-impl.js

```diff
@@ -0,0 +1,25 @@
+"use strict";
+
+const EventTargetImpl = require("../events/EventTarget-impl").implementation;
+
+class PerformanceImpl extends EventTargetImpl {
```

Comment:
My original plan was to make it so that it's possible to directly use the `Performance` class in w3c-hr-time as a webidl2js implementation, but seems like I forgot about `EventTaget` :/ This works well enough I guess.